angular/src/components/settings/vault-timeout-input.component.ts

```diff
@@ -19,6 +19,7 @@ export class VaultTimeoutInputComponent implements ControlValueAccessor, Validat
   }
 
   static CUSTOM_VALUE = -100;
+  static MIN_CUSTOM_MINUTES = 1;
```

Comment:
The default custom timeout value for me is 0 hours and 0 minutes, which makes the error message appear immediately. I think it should default to the minimum allowed time.

Reply:
with the new implementation for the web vault the minimum allowed time should be `1 minute`